language/Portuguese/strings.po

```diff
@@ -1040,1156 +1083,1182 @@ msgid ""
 
 #: src/frontend/mame/ui/auditmenu.cpp:130
 #, c-format
 msgid ""
 "Auditing ROMs for machine %2$u of %3$u...\n"
 "%1$s"
 msgstr ""
+"Auditando ROMs da máquina %2$u de %3$u...\n"
+"%1$s"
 
 #: src/frontend/mame/ui/auditmenu.cpp:142
 msgid "Start Audit"
-msgstr ""
+msgstr "Começar Audit"
 
 #: src/frontend/mame/ui/datmenu.cpp:80
 msgid "Software Usage"
-msgstr ""
+msgstr "Utilização de Software"
 
 #: src/frontend/mame/ui/datmenu.cpp:362
 msgid "Revision: "
-msgstr ""
+msgstr "Revisão: "
 
 #: src/frontend/mame/ui/selmenu.cpp:51
 msgid "Control Panels"
-msgstr ""
+msgstr "Painéis de Controlo"
 
 #: src/frontend/mame/ui/selmenu.cpp:56
 msgid "Artwork Preview"
-msgstr ""
+msgstr "Pré-Visualização da Artwork"
 
 #: src/frontend/mame/ui/selmenu.cpp:60
 msgid "Game Over"
-msgstr ""
+msgstr "Fim de Jogo"
 
 #: src/frontend/mame/ui/selmenu.cpp:69
 msgid "Add or remove favorites"
-msgstr ""
+msgstr "Adicionar ou remover favoritos"
 
 #: src/frontend/mame/ui/selmenu.cpp:70
 msgid "Export displayed list to file"
-msgstr ""
+msgstr "Exportar lista visível para ficheiro"
 
 #: src/frontend/mame/ui/selmenu.cpp:71
 msgid "Show DATs view"
-msgstr ""
+msgstr "Exibir visualização de DATs"
 
 #: src/frontend/mame/ui/selmenu.cpp:237
 msgid "Software part selection:"
-msgstr ""
+msgstr "Selecção de Item de Software:"
 
 #: src/frontend/mame/ui/selmenu.cpp:343
 msgid "BIOS selection:"
-msgstr ""
+msgstr "Selecção de BIOS:"
 
 #: src/frontend/mame/ui/selmenu.cpp:574
 #, c-format
 msgid "Software is clone of: %1$-.100s"
-msgstr ""
+msgstr "O Software é clone de: %1$-.100s"
 
 #: src/frontend/mame/ui/selmenu.cpp:576
 msgid "Software is parent"
-msgstr ""
+msgstr "O Software é original"
 
 #: src/frontend/mame/ui/selmenu.cpp:581
 msgid "Supported: No"
-msgstr ""
+msgstr "Suportado: Não"
 
 #: src/frontend/mame/ui/selmenu.cpp:586
 msgid "Supported: Partial"
-msgstr ""
+msgstr "Suportado: Parcial"
 
 #: src/frontend/mame/ui/selmenu.cpp:591
 msgid "Supported: Yes"
-msgstr ""
+msgstr "Suportado: Sim"
 
 #: src/frontend/mame/ui/selmenu.cpp:596
 #, c-format
 msgid "romset: %1$-.100s"
-msgstr ""
+msgstr "romset: %1$-.100s"
 
 #: src/frontend/mame/ui/selmenu.cpp:612
 #, c-format
 msgid "Driver is clone of: %1$-.100s"
-msgstr ""
+msgstr "O Driver é clone de: %1$-.100s"
 
 #: src/frontend/mame/ui/selmenu.cpp:614
 msgid "Driver is parent"
-msgstr ""
+msgstr "O Driver é original"
 
 #: src/frontend/mame/ui/selmenu.cpp:650
 #, c-format
 msgid "%1$s %2$s"
-msgstr ""
+msgstr "%1$s %2$s"
 
 #: src/frontend/mame/ui/selmenu.cpp:1975
 msgid "Images"
-msgstr ""
+msgstr "Imagens"
 
 #: src/frontend/mame/ui/selmenu.cpp:1976
 msgid "Infos"
-msgstr ""
+msgstr "Infos"
 
 #: src/frontend/mame/ui/selmenu.cpp:2454
 msgid "Usage"
-msgstr ""
+msgstr "Utilização"
 
 #: src/frontend/mame/ui/selmenu.cpp:2489
 msgid "General Info"
-msgstr ""
+msgstr "Info Geral"
 
 #: src/frontend/mame/ui/submenu.cpp:22
-msgid "Miscellaneous Options"
+msgid "Opções Diversas"
```

Comment:
You appear to have accidentally changed the key rather than the translation here.

Reply:
Indeed.
Fixed.